### PR TITLE
Update readme expression

### PR DESCRIPTION
**Pull request by liurupeng**
#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ Design and proposal can be found here: http://bit.ly/k8s-LWS.
...
-- **Topology-aware placement:** Be able to ensure that pods in the same group will be co-located in the same topology.
-- **All-or-nothing restart for failure handling:** Certain ML inference stacks require all pods in the group to be recreated if one pod in the group failed or one container in the pods is restarted.
+- **Dual-template, one for leader and one for the workers:** Supports pod group that is composed of a single leader and a set of workers, and allow to specify a template for the workers and optionally use a second one for the leader pod.
+- **Multiple groups with identical specifications:** Supports creating multiple “replicas” of the above mentioned group. Each group is a single unit for rolling update, scaling, and maps to a single exclusive topology for placement. 
+- **A scale subresource:** A scale endpoint is exposed for HPA to dynamically scale the number replicas (aka number of groups)
+- **Rollout and Rolling update:** Supports performing rollout and rolling update at the group level, which means we upgrade the groups one by one as a unit (i.e. the pods within a group are updated together).
```
Comment (kerthcet) [inline README.md:18]:
```suggestion
- **Rollout and Rolling update:** Supports performing rollout and rolling update at the group level, which means the groups are upgraded one by one as a unit (i.e. the pods within a group are updated together).
```

File: README.md [hunk 2]
```diff
@@ -9,15 +9,15 @@ Design and proposal can be found here: http://bit.ly/k8s-LWS.
...
-- **Multiple groups with identical specifications:** We should be able to create multiple “replicas” of the above mentioned group. Each group is a single unit for rolling update, scaling, and maps to a single exclusive topology for placement. 
-- **A scale subresource:** We will expose a scale endpoint for HPA to dynamically scale the number replicas (aka number of groups)
-- **Rollout and Rolling update:** It will be performed at the group level, which means we upgrade the groups one by one as a unit (i.e. the pods within a group are updated together).
-- **Topology-aware placement:** Be able to ensure that pods in the same group will be co-located in the same topology.
-- **All-or-nothing restart for failure handling:** Certain ML inference stacks require all pods in the group to be recreated if one pod in the group failed or one container in the pods is restarted.
+- **Dual-template, one for leader and one for the workers:** Supports pod group that is composed of a single leader and a set of workers, and allow to specify a template for the workers and optionally use a second one for the leader pod.
```
Comment (kerthcet) [inline README.md:15]:
```suggestion
- **Dual-template, one for leader and one for the workers:** Pod group is composed of a single leader and a set of workers, and allow to specify a template for the workers and optionally use a second one for the leader pod.
```

**Review by kerthcet — COMMENTED** [review]
Generally great.

File: README.md (hunk 2 above)
Comment (ahg-g) [reply README.md:15]: I would actually say: "A replica is a group of a single leader and a ...."

Comment (liurupeng) [reply README.md:15]: updated

**Pull request merged**